modules/ebi-metagenomics/seqtk/seq/main.nf

```diff
@@ -0,0 +1,43 @@
+// This seqtk/seq module is simply copied from the already-existing nf-core module (https://nf-co.re/modules/seqtk_seq/, https://github.com/nf-core/modules/commit/726ee59cd9360a965d96ea9ea8770f16b8ddd6cc)
+// This is because there are not currently any nf-core ways of adding modules from more than one nf-core repo
```

Comment:
it is sadly relevant, the issue isn't seqtk/seq being on nf-core, but because nf-core doesn't allow nf-modules repo to make subworkflows using modules from more than one place, so I had to add it (and fastp) to our ebi-metagenomics nf-modules as a module so I could make the reads_qc subworkflow

The problem is described here: https://github.com/nf-core/tools/issues/1927